OMCompiler/Compiler/NFFrontEnd/NFType.mo

```diff
@@ -1303,6 +1319,19 @@ public
     str := stringAppendList(strl);
   end subscriptedTypeName;
 
+  function addDimensions
```

Comment:
There's already `liftArrayLeftList` that does this.

Reply:
okay i replaced it!